### DISCUSSION TITLE
How do I remap such that I can exit to Normal mode by hitting Capslock rather than 'Ctrl+['?

### DISCUSSION THREAD

**Discussion by anrXiv** [pit-ray/win-vind#274] [General]
I would like to use Capslock to exit to normal mode rather than 'Ctrl+H'.  The escape key doesn't work to exit to normal mode, and I tried using AutoHotKey to remap my Capslock key to 'Esc' and 'Ctrl+[', and neither worked. I can only use Ctrl+[ to exit to normal mode. Is there any way I can change it such that by default I can use Capslock to exit to normal mode?

**Comment by anrXiv** [comment]
Reading the docs, it looks like what I want is to change the default bindings for Edi Normal from "esc-right" or "ctrl-[" to "caps-lock"

Would appreciate any help with this

**Comment by gilbh** [comment]
is this what you are looking for?
```
inoremap <insert> <to_edi_normal>
```

**Reply by anrXiv** [reply]
Yup ended up figuring it out by remapping it in the config, appreciate it though
